File: auth-service/src/main/java/com/course/auth_service/dto/AuthResponse.java
package com.course.auth_service.dto;


public class AuthResponse {
    private String token;
    private String role;
    private String email;
    private Long userId;

    public AuthResponse(String token, String role, String email, Long userId) {
        this.token = token;
        this.role = role;
        this.email = email;
        this.userId = userId;
    }

    public String getToken() { return token; }
    public String getRole() { return role; }
    public String getEmail() { return email; }
    public Long getUserId() { return userId; }
}

File: pathlearn/app/src/main/java/com/pathlearn/pathlearn/data/model/AuthResponse.kt
package com.pathlearn.pathlearn.data.model

import com.google.gson.annotations.SerializedName

data class AuthResponse(
    @SerializedName("token")
    val token: String,

    @SerializedName("role")
    val role: String,

    @SerializedName("email")
    val email: String,

    @SerializedName("userId")
    val userId: Int
)